db/seeds.rb

```diff
@@ -15,6 +15,8 @@
   ENV["FIXTURES_PATH"] = "spec/fixtures"
   ENV["FIXTURES"] = "local_authorities,local_authority_districts,schools"
   Rake::Task["db:fixtures:load"].invoke
+
+  SchoolDataImporterJob.perform_later if School.count < 100
```

Comment:
This would also trigger the job on local development environments. We might want to do it conditionally on review apps only.

Reply:
I've now shifted this into a conditional so only runs in review apps